x-pack/plugins/cross_cluster_replication/__jest__/client_integration/auto_follow_pattern_list.test.js

```diff
@@ -37,6 +37,9 @@ describe('<AutoFollowPatternList />', () => {
   beforeEach(() => {
     server = sinon.fakeServer.create();
     server.respondImmediately = true;
+    // We make requests to APIs which don't impact the UX, e.g. UI metric telemetry,
+    // and we can mock them all with a 200 instead of mocking each one individually.
+    server.respondWith([200, {}, '']);
```

Comment:
Now you understand why this line should have been there first thing when I wrote the test 😊

Reply:
My instinct is that mocking the API route `ui_metric` would be clearer, but I'm willing to try this out. :)